CMakeLists.txt

```diff
@@ -307,7 +312,9 @@ set_target_properties(lib${APP_NAME_LC} PROPERTIES PREFIX "")
 add_dependencies(lib${APP_NAME_LC} libcpluff ffmpeg dvdnav crossguid)
 
 # Codegen
-add_subdirectory(${CMAKE_SOURCE_DIR}/xbmc/interfaces/swig build/swig)
+if(ENABLE_PYTHON)
```

Comment:
Can't you move it cmake/treedata/optional/common/python.txt or is it executed too late then?